composer.json

```diff
@@ -99,6 +99,25 @@
         "psalm:baseline": "@php \"./.phive/psalm\" --set-baseline=psalm.baseline.xml",
         "psalm:cc": "@php \"./.phive/psalm\" --clear-cache"
     },
+    "scripts-descriptions": {
+        "ci": "Runs all dynamic and static code checks.",
+        "ci:composer:normalize": "Checks the formatting and structure of the composer.json.",
+        "ci:dynamic": "Runs all dynamic tests (i.e. currently, the unit tests).",
```

Comment:
I think there should either be a comma each side of "currently", or no comma either side, but not one on one side but not the other.

Reply:
Change in both places and repushed.